examples/safer_autopilot_connect_gateway/README.md

```diff
@@ -0,0 +1,64 @@
+# Safer Autopilot cluster with Connect Gateway
```

Comment:
nit: the naming might be confusing as this is not using the safer cluster module, rather than normal ap module.